src/rf/requirements/workers.txt

```diff
@@ -1,2 +1,3 @@
 GDAL==1.10.0
 numpy==1.10.1
+exifread==2.1.2
```

Comment:
After looking at this further, reprovisioning my app VM fixed this problem.

Which begs the question, why do we need  a separate `workers.txt` requirement file? These are identical: 
  * https://github.com/lliss/raster-foundry/blob/feature/lol/image-metadata/deployment/ansible/roles/raster-foundry.worker/tasks/dependencies.yml   
  * https://github.com/lliss/raster-foundry/blob/feature/lol/image-metadata/deployment/ansible/roles/raster-foundry.app/tasks/dependencies.yml